4_arm.ipynb

```diff
@@ -282,11 +519,22 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "以下のコマンドを実行することで、順運動学によって計算されたエンドエフェクタ('hand_palm_link')の位置姿勢を知ることができます。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "By executing the following command, you can calculate the position and orientation of the end effector ('hand_palm_link') calculated by forward kinematics:"
```

Comment:
"By executing the following command, you can calculate the position and orientation of the end effector ('hand_palm_link') calculated using forward kinematics:"